web/locales/it.json

```diff
@@ -51,7 +55,7 @@
     },
     "tooltips": {
         "carbonintensity": "Intensità di carbonio",
-		"lowcarbon": "Basse emissioni",
+        "lowcarbon": "Basse emissioni",
```

Comment:
Is there a better translation of low carbon? Right now it seems to say low émissions

Reply:
Well, it's not the full phrase, because the full one is too long